tests/fullstack-test2/run.sh

```diff
@@ -32,3 +32,24 @@ docker-compose -f cluster.yaml -f tiflash-dt.yaml exec -T tiflash0 bash -c 'cd /
 
 docker-compose -f cluster.yaml -f tiflash-dt.yaml down
 clean_data_log
+
+docker-compose -f cluster.yaml -f tiflash-dt-sync-grpc.yaml up -d
```

Comment:
Why are so many tests moved to fullstack2? According to the issue, isn’t fullstack taking the shortest time?

Reply:
Because I found that the running test number in fullstack2 is much less than fullstack and tidb-ci
fullstack: 134 tests
fullstack2: 32 tests
tidb-ci: 209 tests
I now move `sync-grpc` test to fullstack.

Reply:
ok